add-on/_locales/en/messages.json

```
@@ -427,6 +427,14 @@
     "message": "Customize which namespaces are logged to Browser Console. Changing this value will trigger extension restart.",
     "description": "An option description for the log level (option_logNamespaces_description)"
   },
+  "option_uploadDir_title": {
+    "message": "MFS Import Directory",
```

We try to avoid "MFS" in user interfaces as it is a technical term and regular users don't know what it means.
```suggestion
    "message": "File Import Directory",
```